frontend/src/Editor/CodeEditor/utils.js

```diff
@@ -172,6 +181,8 @@ function resolveCode(code, customObjects = {}, withError = false, reservedKeywor
     }
   }
 
+  result = checkIfResolvedResultIsFunction(result);
```

Comment:
> **Bito Code Review Agent Run** [#603bf6](https://github.com/ToolJet/ToolJet/pull/10433#issuecomment-2247237815) - 07/24/2024, 08:40 am
> 
> 🔴 High importance
> Issue: The function 'checkIfResolvedResultIsFunction' is called without checking if it is defined or imported. This can lead to runtime errors if the function is not available in the current scope.
> Fix: Ensure that the 'checkIfResolvedResultIsFunction' function is defined or imported in the current file before calling it.
> Code suggestion
> ```
>  @@ -1,6 +1,7 @@
>   import { checkIfResolvedResultIsFunction } from './path/to/module';
>  
>   function resolveCode(code, customObjects = {}, withError = false, reservedKeywords = []) {
>     // existing code
>  +  result = checkIfResolvedResultIsFunction(result);
>     if (withError) return [result, error];
>     return result;
>   }
> ```
> 
> 
> Is this a valid issue, or was it incorrectly flagged by the Agent?
> 
> * [X]  it was incorrectly flagged

Reply:
> **Bito Code Review Agent Run** [#603bf6](https://github.com/ToolJet/ToolJet/pull/10433#issuecomment-2247237815) - 07/24/2024, 08:40 am
> 
> 🔴 High importance
> Issue: The function 'checkIfResolvedResultIsFunction' is called without checking if it is defined or imported. This can lead to runtime errors if the function is not available in the current scope.
> Fix: Ensure that the 'checkIfResolvedResultIsFunction' function is defined or imported in the current file before calling it.
> Code suggestion
> ```
>  @@ -1,6 +1,7 @@
>   import { checkIfResolvedResultIsFunction } from './path/to/module';
>  
>   function resolveCode(code, customObjects = {}, withError = false, reservedKeywords = []) {
>     // existing code
>  +  result = checkIfResolvedResultIsFunction(result);
>     if (withError) return [result, error];
>     return result;
>   }
> ```
> 
> 
> Is this a valid issue, or was it incorrectly flagged by the Agent?
> 
> * [X]  it was incorrectly flagged